releases/release-7.2.0.md

```diff
@@ -0,0 +1,344 @@
+---
+title: TiDB 7.2.0 Release Notes
+summary: Learn about the new features, compatibility changes, improvements, and bug fixes in TiDB 7.2.0.
+---
+
+# TiDB 7.2.0 Release Notes
+
+Release date: xx xx, 2023
+
+TiDB version: 7.2.0
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v7.2/quick-start-with-tidb) | [Installation packages](https://www.pingcap.com/download/?version=v7.2.0#version-list)
+
+7.2.0 introduces the following key features and improvements:
+
+<table>
+<thead>
+  <tr>
+    <th>Category</th>
+    <th>Feature</th>
+    <th>Description</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td rowspan="2">Scalability and Performance</td>
+    <td>Resource groups support <a href="https://docs.pingcap.com/tidb/dev/tidb-resource-control.md#manage-queries-that-consume-more-resources-than-expected-runaway-queries"> managing runaway queries</a> (experimental)</td>
+    <td>Query timeout can now be managed with more granularity, such that different classifications of queries can have different behavior. Queries meeting the threshold can be configurably deprioritized or killed. In the future, this feature will allow for measuring queries by consumption in addition to wall clock time.
```

Comment:
```suggestion
    <td>Query timeout can now be managed with more granularity, such that different classifications of queries can have different behavior. Queries meeting your specified threshold can be deprioritized or killed.
```

Reply:
remove "In the future, this feature will allow for measuring queries by consumption in addition to wall clock time." because it is risky to make future promises. For these future plans, we can add them to https://docs.pingcap.com/tidb/dev/tidb-roadmap.